ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -55,8 +55,7 @@ - (id)initWithDataSource:(id<ADTokenCacheDataSource>)dataSource
     _dataSource = dataSource;
     SAFE_ARC_RETAIN(dataSource);
     
-    _authority = authority;
-    SAFE_ARC_RETAIN(authority);
+    _authority = [authority copy];
```

Comment:
This is too late. I meant to make sure in ADAuthenticationContext that we're not just retaining the passed in input, which we aren't. Doing a copy here is unnecessary.